README.md

```diff
@@ -1,13 +1,13 @@
 ## Ti-Client in Java [Under Construction]
 
-NOTES: For now it works with only TiKV branch disksing/grpc instead of official release.
+NOTES: For now it works only with the disksing/grpc branch of TiKV instead of the official release one.
 
 A Java client for [TiDB](https://github.com/pingcap/tidb)/[TiKV](https://github.com/pingcap/tikv). 
-It suppose to:
+It is supposed to:
 + Communicate via [gRPC](http://www.grpc.io/)
 + Talk to Placement Driver searching for a region
-+ Talk to TiKV reading / writing data the format/encoding as a real TiDB.
-+ Talk Coprocessor for calculation pushdown
++ Talk to TiKV reading/writing data with the format/encoding as a real TiDB.
```

Comment:
How about this? "Talk to TiKV for reading/writing data and the resulted data is encoded/decoded just like what we do in TiDB"